### ISSUE TITLE
Parse PCAP traces to calculate fairness

### ISSUE THREAD

**Issue by ccanel** [ccanel/ns-3-unfair#4]
This is **high** priority.

Since we are building a model to predict the relative fairness of a BBR flow, we need corresponding ground truth labels for our training dataset.

Create a new script called `parse_fairness.py`, based on `parse_pcap.py`, that reads a directory of PCAP traces from different simulations and, for every simulation, outputs a mapping from packet sequence number to some measure of the degree of fairness experienced by the flows at the bottleneck router at the time that the packet departed the bottleneck router. 

Differences from `parse_pcap.py`:
1. This will involve parsing the PCAP trace from the router's output interface instead that from the destination's input interface.
2. The output will be one file per simulation instead of one file for all simulations.
3. An output file will contain an entry for every packet instead of an entry for every simulation.

Later, we will combine these output files with those of `parse_csv.py` (by matching up the sequence numbers) to create our final training data.

Format of PCAP traces: Currently, our simulations capture PCAP traces from (1) src output, (2) router input, (3) router output, and (4) dst input. However, `parse_pcap.py` only looks at (4). `parse_fairness.py` should instead look at (3). The four PCAP traces result in files of this form:
```sh
<details>-0-0.pcap  # (1)
<details>-1-0.pcap  # (2)
<details>-1-1.pcap  # (3)
<details>-2-0.pcap  # (4)
```
See [here](https://github.com/zechengh09/BBR-Receiver/blob/ai/ns-3.27/parse_pcap.py#L190) for an example of selecting one PCAP trace locale.

**Comment by ccanel** [comment]
@zechengh09 Are you interested in working on this? No worries if not, I'm happy to do it.

**Comment by ccanel** [comment]
Resolved by https://github.com/cmu-snap/unfair/pull/2
